tests/test_SE_combine_methods.py

```diff
@@ -0,0 +1,427 @@
+import numpy as np
+import pandas as pd
+import scipy.sparse as sp
+import pytest
+from summarizedexperiment.SummarizedExperiment import SummarizedExperiment
+
+__author__ = "keviny2"
+__copyright__ = "keviny2"
+__license__ = "MIT"
+
+
+rowData1 = pd.DataFrame(
+    {
+        "seqnames": ["chr_5", "chr_3", "chr_2"],
+        "start": [10293804, 12098948, 20984392],
+        "end": [28937947, 3872839, 329837492]
+    },
+    index=["HER2", "BRCA1", "TPFK"],
+)
+colData1 = pd.DataFrame(
+    {
+        "sample": ["SAM_1", "SAM_3", "SAM_3"],
+        "disease": ["True", "True", "True"],
+    },
+    index=["cell_1", "cell_2", "cell_3"],
+)
```

Comment:
- [ ] all SE's seems to have different cell index. can you add a test with overlaps?
- [ ] an example where `rowdata` or `coldata` has no columns but only index

```python
# an example for coldata
colData2 = pd.DataFrame(
    index=["cell_4", "cell_5", "cell_6"],
)
```

-

Reply:
good catch! i'll add test cases for those